packager/mpd/base/period.h

```diff
@@ -117,6 +124,10 @@ class Period {
   // if they contain identical ContentProtection elements. This map is only
   // useful when ContentProtection element is placed in AdaptationSet.
   std::map<std::string, std::list<AdaptationSet*>> adaptation_set_list_map_;
+  // Contains Trickplay AdaptationSets grouped by specific adaptation set
+  // grouping key. These AdaptationSets still have not found reference
+  // AdaptationSet.
+  std::map<std::string, std::list<AdaptationSet*>> trickplay_cache;
```

Comment:
Google C++ style guide requires member variables to end with "_", so rename it to trickplay_cache_.